public/video-ui/src/components/Videos/VideoItem.js

```diff
@@ -3,19 +3,41 @@ import {Link} from 'react-router';
 
 export default class VideoItem extends React.Component {
 
+  renderActiveAssetName() {
+
+    if (this.props.video.activeVersion && this.props.video.assets.length) {
+      const activeAssets = this.props.video.assets.filter((asset) => asset.version === this.props.video.activeVersion)
+      if (activeAssets.length !== -1) {
```

Comment:
does this work? wouldn't `.filter` return an empty array if nothing can be found?